source/_components/sensor.buienradar.markdown

```diff
@@ -61,8 +65,29 @@ Configuration variables:
   - **precipitation_forecast_average**: The average expected precipitation/rain in mm/h within the given timeframe.
   - **precipitation_forecast_total**: The total expected precipitation/rain in mm within the given timeframe. The total expected rain in the configured timeframe will be equal to _precipitation_forecast_total_/_timeframe_ mm/min. So, with timeframe configured to 30 minutes and a value of 5, the expected rain is 5 mm in 30 minutes, which is the same as 10 mm/h. If timeframe is set to 90 minutes and a value of 5, the expected rain is 5 mm in 90 minutes, which is equal to 3.3 mm/h.
   - **irradiance**: Sun intensity in Watt per square meter (W/m2).
-  
-Full configuration example where location is manually specified:
+  - **temperature_1d** [<sup>[1d]</sup>](#1d): The forecasted temperature (in C).
+  - **mintemp_1d** [<sup>[1d]</sup>](#1d): The forecasted minimum temperature (in C).
+  - **rainchance_1d** [<sup>[1d]</sup>](#1d): The forecasted chance for rain (%).
+  - **sunchance_1d** [<sup>[1d]</sup>](#1d): The forecasted chance for sun (%).
+  - **rain_1d** [<sup>[1d]</sup>](#1d): The forecasted amount of rain in mm.
+  - **snow_1d** [<sup>[1d]</sup>](#1d): The forecasted amount of snow in cm.
```

Comment:
Those are the units uses by the underlying weather service.... apparently is is usefull to measure expected snowfall in cm and rain in mm